docs/content/projects/oscal-deep-diff.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "OSCAL Deep Diff"
+description: "OSCAL Deep Diff - compare OSCAL artifacts programmatically and consistently"
+heading: "OSCAL Deep Diff"
+toc:
+  enabled: true
+usabanner: true
+---
+
+OSCAL Deep Diff is an open-source CLI application and library that can produce schema-agnostic comparisons of OSCAL JSON artifacts.
+
+OSCAL Deep Diff was built with the following considerations:
+
+- Document comparisons can vary wildly in intention and desired output, so reliance on an input schema is undesirable. The same tool should be able to compare any OSCAL JSON artifacts.
+- The comparison must be configurable in order to constrain the output document to meet the user's needs. For example, a consumer comparing revisions of a control catalog may not care about capitalization differences in control names between two documents, but a catalog author may care about such differences.
+- Human-readable output formats such as Excel workbooks are appropriate for sharing, but are not appropriate for automated decision making such as a CI/CD system. A document comparison output format should be configurable to match the use case.
+- The tool must be portable and extendable so that it can be integrated into other tools, such as web applications.
+
+## How do I use OSCAL Deep Diff?
+
+OSCAL Deep Diff is currently designed as a command line (CLI) tool which can be used without relying on cloud services. For details on how to install and use OSCAL Deep Diff, see the project page [on GitHub](https://github.com/usnistgov/oscal-deep-diff).
```

Comment:
not sure why the cloud services are mentioned. Any logic? maybe "used locally, without relying on any internet connectivity at the run time"

Reply:
Resolved in [f468d61](https://github.com/usnistgov/oscal-tools/pull/63/commits/f468d61bae90e0ca95dfc1685ee1feeb746bbddf)